test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/get_metrics_incoming_transactions_part_2.feature

```diff
@@ -0,0 +1,659 @@
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: get common metrics in Context Broker
+  As a context broker user
+  I would like get common metrics in Context Broker
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database "orion-test_incomingtransactions_1" in mongo
+  Setup: delete database "orion-test_incomingtransactions_2" in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+############## Metrics to implement:
+#
+#    incomingTransactions: number of requests consumed by the component.
+#    incomingTransactionRequestSize: total size (bytes) in requests associated to incoming transactions (“in” from the point of view of the component).
+#    incomingTransactionResponseSize: total size (bytes) in responses associated to incoming transactions (“out” from the point of view of the component).
+#    incomingTransactionErrors: number of incoming transactions resulting in error.
+#    serviceTime: average time to serve a transaction.
+#    outgoingTransactions: number of requests sent by the component.
+#    outgoingTransactionRequestSize: total size (bytes) in requests associated to outgoing transactions (“out” from the point of view of the component).
+#    outgoingTransactionResponseSize: total size (bytes) in responses associated to outgoing transactions (“in” from the point of view of the component).
+#    outgoingTransactionErrors: number of outgoing transactions resulting in error.
+#
+##############  WARNING:
+#   This tests suite must be executed complete, because the counters are accumulative in the same CB session. If you execute a scenario alone will fail.
```

Comment:
Yes, I think keeping the total counter is useful, however, below this line I explain how to use the reset parameter.